modular/gater/object_handler.go

```diff
@@ -341,7 +341,6 @@ func (g *GateModular) queryResumeOffsetHandler(w http.ResponseWriter, r *http.Re
 	segmentCount, err = g.baseApp.GfSpClient().GetUploadObjectSegment(reqCtx.Context(), objectInfo.Id.Uint64())
 	if err != nil {
```

Comment:
It is best to only ignore the not found error.

Reply:
`if err != nil && err.Error() != metadata.ErrNoRecord.String() {
     return err
}` is more simple.